test/unit_tests/app/test_print_file_builder.py

```diff
@@ -92,6 +92,35 @@ def test_generate_print_row_valid_CE1_IC01(cleanup_test_files):
                                                 '|ONS|123456789|012345678\n')
 
 
+def test_generate_print_row_valid_CE_IC03_1(cleanup_test_files):
+    # Given
+    json_body = json.dumps({
+        "actionType": "CE_IC03_1",
+        "batchId": "1",
+        "batchQuantity": 3,
+        "uac": "test_uac",
+        "caseRef": "test_caseref",
+        "addressLine1": "123 Fake Street",
+        "addressLine2": "Duffryn",
+        "townName": "Newport",
+        "postcode": "NPXXXX",
+        "packCode": "D_ICA_ICLR1",
+        "qid": "31000000001234",
```

Comment:
This really should be a qid of type `21` since it's a individual CE unit

Reply:
Ah sorry, just made it up. Will change now